### ISSUE TITLE
Uncaught SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by pax** [no-chris/chord-symbol#150]
I tried following the [installation instructions](https://github.com/no-chris/chord-symbol#usage), I get `Uncaught SyntaxError: Cannot use import statement outside a module` – both if I clone this repository or I install as node module. Is there any intermediary step I might be missing?

I tried loading it both via `<script src="` and `import {...} from 'path'`

![image](https://user-images.githubusercontent.com/116795/71601340-b9f3c380-2b5b-11ea-8a94-371c3087c16b.png)

**Comment by no-chris** [comment]
Hello,
I actually never tried this way of importing before.
Will try to have a look at the issue this week end.

**Comment by natalyjazzviolin** [comment]
I've got the same issue, would appreciate any advice on this!

**Comment by no-chris** [comment]
I'll try to have a look soon since there seems to have interest for this.

I'm curious though, what is the rationale for importing the lib like this? My thought of this lib was to be used as a low-level component to be compiled in the main bundle, but I might be missing a use case here. Let me know, I'll have a look anyway

**Comment by pax** [comment]
I might be able to shed some light on that. Bundling is a somewhat intermediary/advanced level, not nominally new, but I suspect that the large majority of js wranglers are either unaware or not experienced enough with it. 

I surely weren't one year ago. It still wouldn't be my default / first / obvious thought for prototyping 'see how it works' stage. I'm leaning to think that would be true for other enthusiast- or even intermediary-level devs.

Possibly a first/cheap fix would be an explanatory note in the _Usage_ section.

**Comment by no-chris** [comment]
This will be fixed in the next release, with instructions in the readme on how to do that.

**Comment by no-chris** [comment]
Maybe you could help me test the feature before release?

You would need to check out the following branch locally: 
https://github.com/no-chris/chord-symbol/tree/module-browser-usage

And then create an `html` file at the root of the repository, similar to this one:
```html
<html>
<body>
<script type="module">
	import { chordParserFactory, chordRendererFactory } from './lib/chord-symbol.mjs';

	const parseChord = chordParserFactory();
	const renderChord = chordRendererFactory({ useShortNamings: true});

	const chord = parseChord('C11');
	console.log(renderChord(chord));
</script>
</body>
</html>
```

I tried this locally with Chrome/Firefox/Safari but I would feel better with having someone double-checking it.

Thanks a lot for your assistance

**Comment by pax** [comment]
I get 

> Failed to load module script: The server responded with a non-JavaScript MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec. 
> chord-symbol.mjs:1

on macos 10.12.6 / Apache/2.4.28

**Comment by no-chris** [comment]
Your Apache is probably not configured to handle `.mjs` files, which is something very likely to happen to a lot of users, thanks for uncovering this issue I hadn't thought about 👍 
https://stackoverflow.com/questions/55995339/es6-import-using-mjs-suffix-fails-with-mime-error

So I changed the bundle file name. If you pull the latest changes and replace the `import` statement with the following:
```javascript
import { chordParserFactory, chordRendererFactory } from './lib/chord-symbol-esm.js';
```
it should not throw the error anymore.

**Comment by pax** [comment]
confirmed, it looks like working for me too.

*I googled the error myself, I suspected it was server-side, as it was the same in all browsers, but somehow I didn't stumble upon the simple .htaccess rule

**Comment by no-chris** [comment]
Great, thanks a lot for your help, greatly appreciated.
Will merge the fix into the `1.0.0` release branch.
